src/index.js

```diff
@@ -120,6 +120,7 @@ export type ResizableProps = {
   onResize?: ResizeCallback,
   onResizeStop?: ResizeCallback,
   defaultSize?: Size,
+  scale: number
```

Comment:
Please make it optional and I think it is better to use tuple `[number, number]` for this type.
This is because It is able to set scale individually for X and Y axis 🙏

Reply:
Happy to go with the tuple approach. One comment would be for `react-draggable` the PR I introduced is _not_ a tuple and follows the approach of a unified scale (x and y being identical). Will need to account for that in `react-rnd`, but something that can be handled when that time arrives .

As for the optionality, the reason I didn't make it optional was linting issues. At current, it is technically optional as the value will be set to `1` via the `defaultProps`. Flow will throw an error though as it doesn't have the ability to deduce a value is being passed/set via `defaultProps` (see [this](https://github.com/facebook/flow/issues/3512#issuecomment-406201080) issue).

Can go the route of not using `defaultProps` and setting the value in the `onMouseMove` though if that's desired.

Hopefully that makes sense. Happy to expand further if not 🙂.

Reply:
Hm I see. Let's leave it as it is for now.
LGTM.